src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -68,6 +68,17 @@
             <argument type="service" id="parameter_bag" on-invalid="ignore" />
             <argument type="service" id="logger" on-invalid="ignore" />
             <argument>%kernel.default_locale%</argument>
+            <argument type="collection">
+                <argument type="service" key="get_env">
```

Comment:
`key="env"` (using `get_env()` is weird)

Reply:
Done